llvm/lib/Target/AMDGPU/AMDGPUInstructionSelector.cpp

```diff
@@ -130,6 +158,10 @@ bool AMDGPUInstructionSelector::selectCOPY(MachineInstr &I) const {
   Register DstReg = Dst.getReg();
   Register SrcReg = Src.getReg();
 
+  bool validPhyToRCC = false;
+  if (isCopyPhysicalToVCC(DstReg, SrcReg, *MRI, validPhyToRCC))
+    return validPhyToRCC;
+
   if (isVCC(DstReg, *MRI)) {
```

Comment:
Are you suggesting that an overloaded `isVCC()` should be created with an extra parameter? But where do you call the new version? Replace the call on line 165? Or place it before line 165?